client/ctclient/ctclient.go

```diff
@@ -208,12 +209,9 @@ func getInclusionProofForHash(ctx context.Context, logClient client.CheckLogClie
 	}
 	if sth != nil {
 		// If we retrieved an STH we can verify the proof.
-		verifier := merkletree.NewMerkleVerifier(func(data []byte) []byte {
-			hash := sha256.Sum256(data)
-			return hash[:]
-		})
-		if err := verifier.VerifyInclusionProofByHash(rsp.LeafIndex, int64(sth.TreeSize), rsp.AuditPath, sth.SHA256RootHash[:], hash); err != nil {
-			log.Fatalf("Failed to VerifyInclusionProofByHash(%d, %d)=%v", rsp.LeafIndex, sth.TreeSize, err)
+		verifier := merkle.NewLogVerifier(rfc6962.DefaultHasher)
+		if err := verifier.VerifyInclusionProof(rsp.LeafIndex, int64(sth.TreeSize), rsp.AuditPath, sth.SHA256RootHash[:], hash); err != nil {
+			log.Fatalf("Failed to VerifyInclusionProof(%d, %d)=%v", rsp.LeafIndex, sth.TreeSize, err)
```

Comment:
These could possibly be `log.Exitf` if we don't think the stack dump will be useful to the user.

Reply:
No they can't -- base Go lang `log` package has no `Exitf`